cli/config.go

```diff
@@ -577,9 +578,7 @@ func loadConfiguration(configPath string, ignoreLogging bool) (resolvedConfigPat
 		kataLog.Logger.Level = originalLoggerLevel
 	}
 
-	if tomlConf.Runtime.Tracing {
-		tracing = true
-	}
+	tracing = tomlConf.Runtime.Tracing
```

Comment:
you're modifying something that's not really part of this commit.

Reply:
Because its cyclomatic complexity reaches the top, I must reduce one to add an "if" for netns config.